pkgs/development/node-packages/node-env.nix

```diff
@@ -516,7 +516,7 @@ let
     stdenv.mkDerivation {
       name = "node-shell-${name}-${version}";
 
-      buildInputs = [ python nodejs ] ++ stdenv.lib.optional (stdenv.isLinux) util-linux ++ buildInputs;
+      buildInputs = [ python nodejs ] ++ stdenv.lib.optional (stdenv.isLinux) utillinux ++ buildInputs;
```

Comment:
Why is the script doing this? utillinux is an alias which fails the pipeline. I fixed that and amended it to get this through before we have another merge conflict.